shared/src/TlsVersions.cs

```diff
@@ -100,5 +100,10 @@ public void SetLegacyAcceptableVersions()
             System.Net.ServicePointManager.SecurityProtocol = _net451Protocol;
 #endif
         }
+
+        /// <summary>
+        /// To enable certificate revocation check. Default to be true.
+        /// </summary>
+        public bool CertificateRevocationCheck { get; set; } = true;
```

Comment:
Would you also hook this value up to all the other locations we instantiate an HttpClient (or Mqtt or Amqp connection)?

Reply:
done